src/graphql/helpers.ts

```diff
@@ -21,7 +21,8 @@ export class PublicError extends Error {}
 const ARG_LIMITS = {
   default: {
     first: 1000,
-    skip: 5000
+    skip: 5000,
+    '*_in': 1000 // Default limit for all _in arguments
```

Comment:
This is too high, dont think currently anyone hit this limit, I'd just use 100 here. 
```suggestion
    '*_in': 100 // Default limit for all _in arguments
```